orchestrator/careplanservice/handle_getcareplan.go

```diff
@@ -32,41 +41,58 @@ func (s *Service) handleGetCarePlan(ctx context.Context, id string, headers *fhi
 // if the requester is a participant of one of the returned CareTeams, return the whole bundle, else error
 // Pass in a pointer to a fhirclient.Headers object to get the headers from the fhir client request
 func (s *Service) handleSearchCarePlan(ctx context.Context, queryParams url.Values, headers *fhirclient.Headers) (*fhir.Bundle, error) {
```

Comment:
ouch, this just lists all CarePlans in the CPS? Isn't that going to be terribly slow? I can't think of another approach though... Maybe a custom SearchParameter in future.

Reply:
Well, yes if the user does not provide any params then it will search everything. Ideally I would hope they provide an ID or some other filter.

I suppose we could enforce that the user provides at least something that will limit the results but that sounds messy too